radis/api/exomolapi.py

```diff
@@ -646,83 +646,159 @@ def check_code_level(bdat):
     return None
 
 
-def make_j2b(bdat, alpha_ref_default=0.07, n_Texp_default=0.5, jlower_max=None):
+def make_j2b(
+    bdat, alpha_ref_default=0.07, n_Texp_default=0.5, jlower_max=None, engine="NumPy"
+):
     """compute j2b (code a0, map from jlower to alpha_ref)
 
     Args:
         bdat: exomol .broad data given by exomolapi.read_broad
         alpha_ref_default: default value
         n_Texp_default: default value
         jlower_max: maximum number of jlower
+        engine: "NumPy" (default) for NumPy/Pandas processing, "vaex" for Vaex processing.
```

Comment:
Check in the rest of the code but the term we employ is "pytable" instead of "numpy"